src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -32,7 +32,7 @@ export default function CopyToClipboard({ textToCopy, children }: Props) {
     <Container onClick={handleCopyToClipboard}>
       {isToastMounted && (
         <Toast isActive={isActive}>
-          <Caption color={colors.white}>Copied</Caption>
+          <Caption color={colors.white}>Link Copied</Caption>
```

Comment:
Since we don't have a wallet address in the dropdown, I changed directly the text in the `CopyToClipboard` component. 

Later on, we can make it as a `props` if we have more use cases for this component

Reply:
nice, +1 on making this based on props!